week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -12,13 +12,26 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). 
+   A. console.log(b) as that is not a global variable
+
   2. Remove the line that throws the error.
+  A. Removed 
+
   3. What is printed to the console?
+  A. Value of a,x,d,e
+
   4. How many times is "f1" called?
+  A.  2 times
+
   5. How many times is "f2" called?
+  A. 3 times
+
   6. What value does the "a" parameter take in the first "f1" call?
+  A. value of odd i=1
```

Comment:
7. `a` is initially 6 but is then incremented by `1` on each iteration. Think about what value `a` will have incremented to when `f1` is called for the first time.